src/main.py

```diff
@@ -15,6 +16,8 @@
     # NOTE: This *has* to be QGuiApplication otherwise the system tray right-click menu won't show
     app = QGuiApplication(sys.argv)
 
+    QQuickStyle.setStyle("Basic")
```

Comment:
Will need to add a comment above this line of why it's there (mostly as I'll probably forget otherwise haha). I'll open a PR within a week unless ofc you beat me to it 😉

<details>

https://forum.qt.io/topic/145229/qt-6-5-the-current-style-does-not-support-customization-of-this-control

https://stackoverflow.com/questions/76625756/the-current-style-does-not-support-customization-of-this-control

</details>